apps/hosts/urls.py

```diff
@@ -12,5 +12,6 @@
         name='get_challenge_host_list'),
     url(r'challenge_host_team/(?P<challenge_host_team_pk>[0-9]+)/challenge_host/(?P<pk>[0-9]+)$',
         views.challenge_host_detail, name='get_challenge_host_details'),
-
+    url(r'remove_challenge_host/(?P<challenge_host_team_pk>[0-9]+)$',
```

Comment:
Forgot to mention that please change the prefix of url to `remove_self_from_challenge_host_team`?